### PR TITLE
ggv2: cleanup logs

### PR DESCRIPTION
**Pull request by yuval-k**
cleanup spurious logs - so that in steady state no logs are emitted

### CODE REVIEW

File: projects/gloo/pkg/api/converters/kube/opaque_secret.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func (c *OpaqueSecretConverter) FromKubeSecret(ctx context.Context, rc *kubesecr
 
 	// let the ResourceClient try to unmarshal the opaque secret into one of the Gloo secret types
 	resource, err := rc.FromKubeSecret(secret)
-	if err != nil {
+	if err != nil && err != kubesecret.NotOurResource {
```
Comment (nfuden) [inline projects/gloo/pkg/api/converters/kube/opaque_secret.go:29]: hilarious fix

File: projects/gateway2/proxy_syncer/kube_gw_translator_syncer.go [hunk 2]
```diff
@@ -112,23 +112,13 @@ func (s *ProxyTranslator) syncXds(
...
-	snap *xds.EnvoySnapshot,
 	proxyKey string,
 	reports reporter.ResourceReports,
 ) error {
 	ctx = contextutils.WithLogger(ctx, "kube-gateway-xds-syncer")
 	logger := contextutils.LoggerFrom(ctx)
-	logger.Infof("begin kube gw sync for proxy %s (%v listeners, %v clusters, %v routes, %v endpoints)",
-		proxyKey, len(snap.Listeners.Items), len(snap.Clusters.Items), len(snap.Routes.Items), len(snap.Endpoints.Items))
-	stopwatch := statsutils.NewTranslatorStopWatch("sync-xds-and-gloo-status")
-	stopwatch.Start()
```
Comment (nfuden) [inline projects/gateway2/proxy_syncer/kube_gw_translator_syncer.go:124]: ?

Comment (nfuden) [reply projects/gateway2/proxy_syncer/kube_gw_translator_syncer.go:124]: We force sync alot and writing reports isnt that interesting as the force sync breaks the actual metric

**Review by nfuden — APPROVED** [review]

**Review by lgadban — APPROVED** [review]

**Pull request merged**